internal/controllers/user_devices_controller.go

```diff
@@ -1453,6 +1457,12 @@ func (udc *UserDevicesController) PostMintDevice(c *fiber.Ctx) error {
 		return opaqueInternalError
 	}
 
+	cid, err := udc.ipfsSvc.UploadImage(c.Context(), imageData)
+	if err != nil {
+		udc.log.Err(err).Msg("failed to upload NFT image to IPFS while minting")
+	}
+	userDevice.IpfsImageCid = null.StringFrom(cid)
```

Comment:
Do you still want to do this if the above fails?

Reply:
oo nope.. I thought that having an empty string would make valid = false, just tested it  to double check and turns out that is not the case